common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -905,13 +909,31 @@ def test_user_can_create_new_team_successfully(self):
         And the number of teams should be updated on the topic card
         And if I switch to "My Team", the newly created team is displayed
         """
+
```

Comment:
Nit: why this blank line?

Another nit: document the expected event in the docstring.